src/isomorphic/children/ReactChildren.js

```diff
@@ -146,7 +146,7 @@ function countChildren(children, context) {
 }
```

Comment:
Yeah, that's true. You can return a collection from your mapper which means we need to refactor the logic in order to prefix those keys correctly…